### DISCUSSION TITLE
geomap: allow geojson data coming from a rest api or db etc #51347

### DISCUSSION THREAD

**Discussion by yosiasz** [grafana/grafana#51348] [General]
Greetings!

Would be nice if geomap allowed geojson coming not just from maps/files or URL but also from another data source  query, just as you can do it for lon/lat

Right now I can do the following by using my own rest api

<img width="231" alt="image" src="https://user-images.githubusercontent.com/900951/175378833-624f17eb-873c-4263-8cf2-de79aed93bce.png">

But I want to do this from another data source and plugin the geojson column returned to plug it in as a value

![image](https://user-images.githubusercontent.com/900951/175379344-84cc32fc-549c-4004-8adf-6c453399afb2.png)

Just as I can enter/select long lat from data source query

<img width="200" alt="image" src="https://user-images.githubusercontent.com/900951/175379593-1cd9acf1-cd4e-42b8-84cd-961525c490be.png">

Thanks for lovely/powerful grafana

**Comment by natellium** [comment]
Hey there! We want to let you all know that we are moving to Github Issues for feature requests. You can read more about the move https://github.com/grafana/grafana/discussions/73424.

I will be closing this discussion, but if you’re still interested in this request being implemented please don’t hesitate to open a [feature request](https://github.com/grafana/grafana/issues/new?assignees=&labels=&projects=&template=1-feature_requests.md). Sorry for the inconvenience, and thanks for your contributions!
